ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -314,7 +325,10 @@ static VirtualizationRootHandle FindRootAtVnode_Locked(vnode_t vnode, uint32_t v
         
         if (rootEntry.rootVNode == vnode && rootEntry.rootVNodeVid == vid)
         {
-            assert(fileId.fsid.val[0] == rootEntry.rootFsid.val[0]);
+            assertf(
```

Comment:
Will these asserts trigger a KP in release builds, or are they only in effect for debug builds?

Reply:
They are currently active in release builds too, the macro controlling this is `MACH_ASSERT=1`.

Reply:
Great thanks!